### PR TITLE
chore(playground): update resvg-wasm

### PR DESCRIPTION
**Pull request by jeetiss**
- updates `resvg-wasm`
- uses wasm files from node_modules as bin files

### CODE REVIEW

**Comment by jeetiss** [comment]
it should work when https://github.com/yisibl/resvg-js/pull/186 will be released

**Comment by yisibl** [comment]
Thanks!

**Comment by jeetiss** [comment]
@yisibl looks like something is wrong with bindings. it is built, but i get an error in console

```
Uncaught (in promise) LinkError: WebAssembly.instantiate(): Import #0 module="wbg" function="__wbg_new_15d3966e9981a196" error: function import requires a callable
```

**Comment by yisibl** [comment]
Try this? https://github.com/yisibl/resvg-js/pull/196

**Comment by jeetiss** [comment]
> Try this? https://github.com/yisibl/resvg-js/pull/196

It works 🚀

**Comment by yisibl** [comment]
I just released 2.3.1 and ran your branch locally and the error was resolved.

File: playground/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "@babel/runtime": "^7.19.0",
     "@monaco-editor/react": "^4.4.5",
     "@resvg/resvg-js": "^1.4.0",
```
Comment (yisibl) [inline playground/package.json:14]: Can this be upgraded to the latest version?

Comment (jeetiss) [reply playground/package.json]: playground doesn't use this package, so i remove it

Comment (yisibl) [reply playground/package.json]: OK

**Pull request merged**